app/controllers/datasets_controller.rb

```diff
@@ -20,16 +28,17 @@ def show
 
   def edit
     @dataset = Dataset.find(params[:id])
+    work = Work.find(@dataset.work_id)
+    @datacite = Datacite::Resource.new_from_json(work.data_cite)
```

Comment:
FUTURE UPDATE: I am thinking that our current database structure with a `Work` and a `Dataset` table might be adding extra complexity to our code. 

I am thinking that we should ditch the `Dataset` table (since it has a 1-1 relationship with the Work) and just keep the `Work` table. We could move the few fields in the Dataset table to the Work table and keep the code simpler.

Reply:
+1 for this idea.